### ISSUE TITLE
links in table headers are black on purple

### ISSUE THREAD

**Issue by jyesberg** [jyesberg/SPF2.0#13]
Anchors in thead>th cells are black font on purple background, foreground/background contrast ratio too low

**Comment by jyesberg** [comment]
Fixed in Jul 25 commit. See https://github.com/jyesberg/SPF2.0/commit/b2caefe5db156c2213e8cc7e98153c07bc4401c8#diff-84aecf132923ba3ea7ed457553816058
